src/main/java/org/jenkinsci/test/acceptance/po/Job.java

```diff
@@ -369,7 +371,7 @@ public Build getLastBuild() {
     public <T extends Parameter> T addParameter(Class<T> type) {
         ensureConfigPage();
 
-        control("/properties/hudson-model-ParametersDefinitionProperty/specified").check();
+        control(by.checkbox("This project is parameterized")).check();
```

Comment:
need a checkbox instance here so we don't unclick this, other places I took a shortcut so didn't need to add the text. (of course other ways to do this)